File: 백준/Bronze/11720. 숫자의 합/숫자의 합.cc
#include <iostream>

using namespace std;

int n, ans;
string num;

int main() {
  cin >> n;
  cin >> num;
  ans = 0;
  for(int i = 0; i < n; i++) {
    ans += num[i] - '0';
  }
  cout << ans << '\n';
  return 0;
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());
        String num = br.readLine();
        int ans = 0;
        for(int i = 0; i < n; i++) {
        	ans += num.charAt(i) - '0';
        }
        System.out.println(ans);
        br.close();
    }
}